src/Testcontainers/Containers/ResourceReaper.cs

```diff
@@ -61,7 +63,7 @@ private ResourceReaper(Guid sessionId, IDockerEndpointAuthenticationConfiguratio
         .WithCleanUp(false)
         .WithExposedPort(RyukPort)
         .WithPortBinding(TestcontainersSettings.ResourceReaperPublicHostPort.Invoke(dockerEndpointAuthConfig), RyukPort)
-        .WithBindMount("/var/run/docker.sock", "/var/run/docker.sock", AccessMode.ReadOnly)
+        .WithBindMount(dockerSocketFilePath, "/var/run/docker.sock", AccessMode.ReadOnly)
```

Comment:
@eddumelendez @kiview @mdelapenya Does the socket always bind to `/var/run/docker.sock` (destination)? Is that similar to your language implementation?

Reply:
It is the same as in Java, see: https://github.com/testcontainers/testcontainers-java/blob/cae12726c04ca80a7049b49245b8d005ac38e95d/core/src/main/java/org/testcontainers/utility/RyukContainer.java#L23

Reply:
Thanks